services/merak-topo/handler/topo_deploy.go

```diff
@@ -52,12 +52,13 @@ var (
 	}
 )
 
-func CreateTopologyClasses(client dynamic.Interface, name string, links []map[string]interface{}) error {
+func CreateTopologyClasses(client dynamic.Interface, name string, links []database.Vlink) error {
 	rc := NewTopologyClass(name, links)
 
 	_, err := client.Resource(topologyClassGVR).Namespace(namespace).Create(Ctx, rc, metav1.CreateOptions{})
 
-	log.Printf("Creating TopologyClass %s", name)
+	// log.Printf("Creating TopologyClass %s", name)
+	// log.Printf("topology class details %s", rc)
```

Comment:
@zzengcs Remove the commented-out lines.

Reply:
addressed